elkscmd/basic/host-ibmpc.c

```diff
@@ -0,0 +1,359 @@
+/*
+ * Architecture Specific routines for CGA
+ * Sep 2024 Takahiro Yamada
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#if __ia16__
+#include <arch/io.h>
+#else
+#define inb(port)           0
+#define inw(port)           0
+#define outb(value,port)
+#define outw(value,port)
+#endif
```

Comment:
Remove all the "faked" defines and the `#if __ia16__` above it, since the definitions are inaccurate and don't work. This file does not support compilation with Watcom, which uses host-stubs.c instead.